contracts/PushCore/PushCoreV3.sol

```diff
@@ -768,21 +763,22 @@ contract PushCoreV3 is
     }
 
     /// @inheritdoc IPushCoreV3
-    function handleChatRequestData(address requestSender, address requestReceiver, uint256 amount) public {
+    function handleChatRequestData(address requestSender, address requestReceiver, uint256 amount) external {
```

Comment:
**Critical Issue:**

1. The function is external and is supposed to be capable of handling chatReq on Eth chain by any user. However, it wrongly includes a check that only Comm Contract can call it. That's invalid
2. Validation on requestReceiver address is needed, else tokens get locked for an invalid address.
3. Validation on amount passed is needed as amount arg should be non-zero.

cc: @0xNilesh @Zartaj0